routes/api.php

```diff
@@ -96,6 +96,46 @@
         });
     });
 
+    Route::group(['prefix' => 'admin', 'middleware' => 'admin'], function () {
```

Comment:
is all this also needed for the head teacher?

Reply:
.... yes.